webrtc.html

```diff
@@ -1345,6 +1346,12 @@
               </li>
 
               <li>
+                <p>If <var>candidate</var> is missing values for both <var>sdpMid</var> and
+                <var>sdpMLineIndex</var>, reject <var>p</var> with a <code>TypeError</code>
+                and jump to the step labeled <em>Return</em>.</p>
```

Comment:
This is just a formatting thing. A nice way to handle error cases without having to jump to a "return label" is to express it like: If bad condition is true, return a promise rejected with BadThingError. Then the steps bails at that point and you can define p much later in the steps. See example in the promise writing guide [1].

[1] https://www.w3.org/2001/tag/doc/promises-guide#example-validated-delay